internal/common/common_unitest_db.go

```diff
@@ -77,18 +78,26 @@ func TerminateDBTest() {
 	gDbCtx.pool = nil
 }
 
-func PrepareTestDB(dbName string, extrasSchemas ...interface{}) *gorm.DB {
+// Creates a valid postgresql db name from a random uuid
+// DB names (and all identifiers) must begin with a letter or '_'
+// Additionally using underscores rather than hyphens reduces the chance of quoting bugs
+func randomDBName() string {
+	return fmt.Sprintf("_%s", strings.ReplaceAll(uuid.New().String(), "-", "_"))
+}
+
+func PrepareTestDB(extrasSchemas ...interface{}) (*gorm.DB, string) {
+	dbName := randomDBName()
 	dbTemp, err := gorm.Open("postgres", fmt.Sprintf("host=127.0.0.1 port=%s user=admin password=admin sslmode=disable", gDbCtx.GetPort()))
 	Expect(err).ShouldNot(HaveOccurred())
 	defer dbTemp.Close()
 
-	dbTemp = dbTemp.Exec(fmt.Sprintf("CREATE DATABASE %s;", strings.ToLower(dbName)))
+	dbTemp = dbTemp.Exec(fmt.Sprintf("CREATE DATABASE %s;", dbName))
 	Expect(dbTemp.Error).ShouldNot(HaveOccurred())
 
 	db, err := gorm.Open("postgres",
-		fmt.Sprintf("host=127.0.0.1 port=%s dbname=%s user=admin password=admin sslmode=disable", gDbCtx.GetPort(), strings.ToLower(dbName)))
+		fmt.Sprintf("host=127.0.0.1 port=%s dbname=%s user=admin password=admin sslmode=disable", gDbCtx.GetPort(), dbName))
 	Expect(err).ShouldNot(HaveOccurred())
-	// db = db.Debug()
+	db = db.Debug()
```

Comment:
I think that it will span stdout

Reply:
Yeah, sorry didn't mean any of this to be merged this way.